README.md

```diff
@@ -35,21 +32,23 @@ Current release info
 | Name | Downloads | Version | Platforms |
 | --- | --- | --- | --- |
 | [![Conda Recipe](https://img.shields.io/badge/recipe-dask-green.svg)](https://anaconda.org/conda-forge/dask) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/dask.svg)](https://anaconda.org/conda-forge/dask) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/dask.svg)](https://anaconda.org/conda-forge/dask) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/dask.svg)](https://anaconda.org/conda-forge/dask) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-dask--core-green.svg)](https://anaconda.org/conda-forge/dask-core) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/dask-core.svg)](https://anaconda.org/conda-forge/dask-core) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/dask-core.svg)](https://anaconda.org/conda-forge/dask-core) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/dask-core.svg)](https://anaconda.org/conda-forge/dask-core) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-distributed-green.svg)](https://anaconda.org/conda-forge/distributed) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/distributed.svg)](https://anaconda.org/conda-forge/distributed) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/distributed.svg)](https://anaconda.org/conda-forge/distributed) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/distributed.svg)](https://anaconda.org/conda-forge/distributed) |
 
-Installing dask
-===============
+Installing dask-split
+=====================
 
-Installing `dask` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+Installing `dask-split` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
```

Comment:
Even if we need a name for this recipe of outputs, I don't think `dask-split` should be referenced anywhere.

Reply:
Yeah this text is autogenerated. Not too much we can do here.